native/src/utils/getCachedThumbnail.ts

```diff
@@ -0,0 +1,22 @@
+import { Platform } from 'react-native'
+
+import { URL_PREFIX } from '../constants/webview'
+import { PageResourceCacheStateType } from './DataContainer'
+
+const getCachedThumbnail = (thumbnail: string, resourceCache: PageResourceCacheStateType | undefined): string => {
+  const resource = resourceCache?.[thumbnail]
+
+  if (resource) {
```

Comment:
:wrench: You can changes this to 
```
if(!resource){
return thumbnail}
}

    if (Platform.OS === 'ios') {
      return `~${cachedThumbnail.substring(cachedThumbnail.indexOf('/Documents'))}`
    }
    return `${URL_PREFIX}${cachedThumbnail}`
```

Makes it easier to understand, as the nested conditional is avoided.